snoop/data/analyzers/tika.py

```diff
@@ -71,7 +70,9 @@ def call_tika_server(endpoint, data):
 @returns_json_blob
 def rmeta(blob):
     with blob.open() as f:
+        tracer.current_span().add_annotation('call tika')
```

Comment:
Any version works and provides practically the same data. Annotations allow one to display more events in the same view.

Reply:
This is about exceptions and errors. I guess the tracer will catch those with the context manager. We'll annotate the start and end events under `span` instead of `current_span()`, so you get the same data on the UI.

Reply:
Please check the tasks.py file, for all exceptions an annotation is created.